### PR TITLE
Update Clevertap minimal app to utilize 15.3.4 of Android Point SDK

### PR DESCRIPTION
**Pull request by duncan-lau**

### CODE REVIEW

**Comment by nehabluedot** [comment]
fcm push was not recieved on device while testing

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+.idea
+.idea/*
+.gradle
+/local.properties
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+google-services.json
+.gradle/
```
Comment (poborin) [inline .gitignore]: why not gitignore.io? https://www.toptal.com/developers/gitignore?templates=android,androidstudio,macos

**Review by poborin — APPROVED** [review]

**Comment by duncan-lau** [comment]
> fcm push was not recieved on device while testing

@nehabluedot please try to test FCM with Test Push notification from Clevertap Dashboard for now, thx!

File: app/src/main/java/au/com/bluedot/clevertapdemoapp/MainApplication.java [hunk 2]
```diff
@@ -26,6 +30,7 @@
 import au.com.bluedot.point.net.engine.BDError;
 import au.com.bluedot.point.net.engine.BeaconInfo;
```
Comment (nehabluedot) [inline app/src/main/java/au/com/bluedot/clevertapdemoapp/MainApplication.java]: unused imports can be removed

Comment (duncan-lau) [reply app/src/main/java/au/com/bluedot/clevertapdemoapp/MainApplication.java]: Done!

**Review by nehabluedot — APPROVED** [review]

**Pull request merged**